idea-valley/Kandidatas.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+class Kandidatas extends IdeaValley
+{
+    public function run($kandidatas)
+    {
+        $sugebejimai = array(
+            'PHP 5.5',
```

Comment:
Leave should should use 1 instead of 2:
1. Start using PHP 5.5 (namespaces, short array syntax http://docs.php.net/manual/en/language.types.array.php#example-96)
2. Write **PHP 4** in requirements (I suggest You this considering your codestyle)